src/igvfd/tests/data/inserts/construct_library_set.json

```diff
@@ -42,9 +42,7 @@
         "selection_criteria": [
             "accessible genome regions"
         ],
-        "genes": [
-            "0c9a9a6f-02de-4072-a5a3-6628f13070f6"
-        ],
+        "large_scale_gene_list": "/tabular-files/igvf:tabular_file_01_bed",
```

Comment:
Same thing with this insert. The inserts only fail to load with "large_scale_gene_list" and "large_scale_loci_list" and since these share the same dependencies as small_scale_gene_list/small_scale_loci_list and genes/loci I don't think it is a dependency issue however when I remove all the dependencies the inserts do load fine so I am so confused how to get this to load properly.

Reply:
Also I've introduced tests which patch with the large_scale_* properties and I've patched on a local demo with no problems so I think it's an insert issue.

Reply:
Posted an object identical to the insert failing to load on a local demo and it posts fine:
<img width="1213" alt="Screenshot 2024-01-04 at 2 23 48 PM" src="https://github.com/IGVF-DACC/igvfd/assets/78115078/dd6b9fb5-5ca6-4f1a-af0b-2f9395fd65bc">
so I definitely think it's an insert loading issue.

Reply:
Posted a construct library set identical to the insert failing to load and it posted fine, so I think it's definitely an insert loading issue.
<img width="934" alt="Screenshot 2024-01-04 at 2 25 37 PM" src="https://github.com/IGVF-DACC/igvfd/assets/78115078/1f8146d9-4c56-4729-aa36-302411f1aec8">

Reply:
So the problem with loading the inserts is that the pipelines for loading inserts removed the large_scale_* properties, but the property is required due to the scope dependency. Removing scope as well in the pipeline allows prediction_set inserts to load properly, but construct library set inserts do not since scope is a required property for this type. I don't think we can put large_scale_* in the inserts for construct library sets without modifying the code for loading inserts, so I've just left the property out of the inserts for this type. I have tested the properties otherwise. If the reviewer can think of another way let me know...